src/client/options/mod.rs

```diff
@@ -552,6 +552,16 @@ pub struct ClientOptions {
     #[builder(default)]
     #[cfg(test)]
     pub(crate) heartbeat_freq_test: Option<Duration>,
+
+    /// Allow use of the `load_balanced` option.
+    // TODO RUST-653 Remove this when load balancer work is ready for release.
+    #[builder(default, setter(skip))]
+    #[serde(skip)]
+    pub(crate) allow_load_balanced: bool,
```

Comment:
This awkward double-flag construction allows `load_balanced` to be populated by the URI/TXT parsing as it will be  on release, but prevents that from being used outside of tests that have access to set `allow_load_balanced` to be true.